cli/cmd/configure_monitoring.go

```diff
@@ -37,7 +37,15 @@ var allowedMonitoringTypes = []string{
 var monitoringCmd = &cobra.Command{
 	// Use:          "monitoring <monitoring_provider> --project=<project> --service=<service> --service-indicators=<service_indicators_file_path> --service-objectives=<service_objectives_file_path> --remediation=<remediation_file_path>",
 	Use:          "monitoring <monitoring_provider> --project=<project> --service=<service>",
-	Short:        "Configures monitoring",
+	Short:        "Configures monitoring provider",
+	Long: `Configure a monitoring solution for a Keptn cluster. This command sets up Dynatrace or Prometheus monitoring in case it is not installed yet. 
```

Comment:
The next has been copied from https://keptn.sh/docs/0.6.0/reference/cli/#keptn-configure-monitoring .

As far as I know, configure monitoring also sets up dynatrace and prometheus to some extent. I can remove the *setup* text, if that's okay for everyone.

Reply:
I think Andreas' concern is that a user might think that this command deploys the `dynatrace-service` and `dynatrace-sli-service`, but this is not the case. Actually, the `dynatrace-service` has to be deployed before running this command. 

Instead of removing the information, my suggestion is: 
`Configure a monitoring solution for a Keptn cluster. This command sets up Dynatrace or Prometheus monitoring if it is not installed. Before executing the command, the dynatrace-service or prometheus-service has to be deployed.`

Reply:
Thanks! Updated.